src/Entity/EntityTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Acquia\LiftClient\Entity;
+
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Normalizer\CustomNormalizer;
+use Symfony\Component\Serializer\Serializer;
```

Comment:
Any chance that we can remove dependency to Symfony (without drowning ourselves)? 

Not required right away though, since we are mainly on D8 right now, but D7 is upcoming, and later non-Drupal use cases too.